package com.example.challengeaccepted.platform.data.network.exception

open class ErrorException(val msg: Int = -1) : Exception()